generator/moreschemavalidation_fixtures_test.go

```diff
@@ -39,19 +39,19 @@ func initFixture2116() {
 	flattenRun := f.AddRun(false).WithMinimalFlatten(true)
 
 	flattenRun.AddExpectations("case1_fail_omitempty_false_not_hoisted_by_ref.go", []string{
-		"Body *ObjectWithOmitemptyFalse `json:\"Body\"`",
+		"Body *ObjectWithOmitemptyFalse `json:\"Body\" xml:\"Body\"`",
```

Comment:
@nubenum you would save yourself a lot of pain updating all the model tests by making the systematic addition of XML tags opt-in. I _do_ think this is a nice addition, but making it appear everywhere ,every time is simply causing a lot of impacts.

Reply:
This is a good idea. However, I have decided to reuse the --default-produces and --default-consumes flags for that. If either of them is set to xml, xml struct tags will be added. What do you think, do you prefer a separate flag `--with-xml`?